template.php

```diff
@@ -979,6 +979,7 @@ function europa_preprocess_node(&$variables) {
     $variables['content_column_class'] = 'col-md-12';
   }
 
+  $variables['site_name'] = variable_get('site_name');
```

Comment:
in a nexteuropa instance that variable is always set. what would be in your opinion a "default" value for this? :)